src/main/java/org/terasology/logic/health/event/ChangeMaxHealthEvent.java

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.logic.health.event;
+
+import org.terasology.entitySystem.event.Event;
+
+/**
+ * This Event is sent out whenever a system wants to alter the maxHealth of an entity.
+ */
+public class ChangeMaxHealthEvent implements Event {
```

Comment:
IMO it makes sense if other entities are able to "join the discussion", so consider making this an `AbstractValueModifiableEvent` and adjust the Health-internal receiver to be the last entity reacting to this event (see comments above)